test/html2react.test.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { renderToStaticMarkup } from 'react-dom/server';
+
+import renderer from 'react-test-renderer';
+
+import HTML2React from '../src';
+
+const dataset = [
+  {
+    input: "<p>Hello!</p>",
+    output: '<p>Hello!</p>',
+  },
+  {
+    input: "<input type='checkbox' checked/>",
+    output: '<input type="checkbox" checked=""/>',
```

Comment:
I prefer `checked` without `=""`, but it's what ReactDOMServer does and it just works.